examples/axum-key-value-store/src/main.rs

```diff
@@ -1,3 +1,8 @@
+fn main() {
+    eprintln!("this example has not yet been updated to hyper 1.0");
```

Comment:
Why not? I did it with little effort for the `tower-async` version. Want me to contribute this as a PR to your PR?

Reply:
Same reason as https://github.com/tower-rs/tower-http/pull/348#discussion_r1399815537, axum doesn't yet support http 1.0

Reply:
True. For now I was using your branch. I get you might not want to do that, but now you do not have an example at all... so dunno.

Reply:
> axum doesn't yet support http 1.0

but [now it does](https://tokio.rs/blog/2023-11-27-announcing-axum-0-7-0), right?

Reply:
Yep it does.

Reply:
A PR to restore this example using axum 0.7: https://github.com/tower-rs/tower-http/pull/448